src/java.base/share/classes/java/util/SequencedMap.java

```diff
@@ -257,45 +257,56 @@ default V putLast(K k, V v) {
     }
 
     /**
-     * Returns a {@link SequencedSet} view of this map's keySet.
+     * Returns a {@code SequencedSet} view of this map's {@link #keySet keySet}.
      *
      * @implSpec
-     * The implementation in this interface returns a {@code SequencedSet}
-     * implementation that delegates all operations either to this map or to this map's
-     * {@link #keySet}, except for its {@link SequencedSet#reversed reversed} method,
-     * which instead returns the result of calling {@code sequencedKeySet} on this map's
-     * reverse-ordered view.
+     * The implementation in this interface returns a {@code SequencedSet} instance
+     * that behaves as follows. Its {@link SequencedSet#add add} and {@link
+     * SequencedSet#addAll addAll} methods throw {@link UnsupportedOperationException}.
+     * Its {@link SequencedSet#reversed reversed} method returns the {@link
+     * #sequencedKeySet sequencedKeySet} view of the {@link #reversed reversed} view of
+     * this map. Each of its other methods calls the corresponding method of the {@link
```

Comment:
Hello Stuart, should this (and other similar places in this PR) instead be:
```
Each of its other methods call
```
call instead of calls?

Reply:
"Each" is usually considered singular so "Each ... calls" is proper. It looks weird though since the plural "methods" is right next to "calls."